spec/rspec/matchers/built_in/change_spec.rb

```diff
@@ -223,7 +223,7 @@ def ==(other)
   it "warns if passed a block using do/end instead of {}" do
     expect do
       expect {}.to change do; end
-    end.to raise_error(SyntaxError, /block passed to the `change` matcher/)
+    end.to raise_error(ArgumentError, /Block not received by the `change` matcher/)
```

Comment:
I think this should still be a `SyntaxError` it's not a Ruby syntax error but it is an RSpec one.